mw-webapp/src/service/auth/writeNewUserCredentials.ts

```diff
@@ -1,21 +1,19 @@
-import {getRedirectResult} from "firebase/auth";
-import {auth} from "src/firebase";
+import {User} from "firebase/auth";
 import {UserService} from "src/service/UserService";
 
 /**
  * Create new user on firebase Users collection after google login
  */
-export const writeNewUserCredentials = async () => {
-  const userCredentials = await getRedirectResult(auth);
-  if (!userCredentials) {
+export const writeNewUserCredentials = async (currentUser: User) => {
+  if (!currentUser) {
     return;
   }
-  if (userCredentials.user.email && userCredentials.user.displayName) {
+  if (currentUser.email && currentUser.displayName) {
```

Comment:
Does displayName required? If it is optional why can't we register user without it?

Reply:
Why do you check it than?

Reply:
Deleted check and added default value if displayName or email is not exist